app/(home)/home/components/Carousel.tsx

```diff
@@ -125,7 +125,7 @@ function BottomIndicator(props: BottomIndicatorProps) {
   const { index, amount, onClick } = props;
 
   return (
-    <div className="flex flex-row absolute top-4 right-4 bg-[#FFF8FF] p-2 rounded-r-full rounded-bl-full">
+    <div className="flex flex-row absolute md:top-4 md:right-4 md:bg-[#FFF8FF] p-2 rounded-r-full rounded-bl-full top-[84%] right-[36%] bg-[#fef7ffb3]">
```

Comment:
```suggestion
    <div className="flex flex-row absolute md:top-4 md:right-4 md:bg-[#FFF8FF] p-2 rounded-r-full rounded-bl-full max-md:bottom-4 right-1/2 max-md:transform max-md:translate-x-1/2 bg-[#fef7ffb3]">
```
O indicador do carousel não está centralizado:
<img width="758" alt="image" src="https://github.com/davidgaspardev/web-acto/assets/37712275/eea3dfaf-371e-4ec9-bdaa-2b9a831798d0">

Para centralizar elementos com **position**, não é bom usar porcentagem nas coordenadas (`top`, `right`, `bottom` e `left`), mas sim transform com translate (`X` ou `Y`).